libs/data-mapper-v2/src/components/functions/Panel.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState, useMemo } from 'react';
+import { useStyles } from './styles';
+import { ChevronDoubleRightRegular, ChevronDoubleLeftRegular } from '@fluentui/react-icons';
+import { useIntl } from 'react-intl';
+import { InlineDrawer } from '@fluentui/react-components';
+import { SearchBox } from '@fluentui/react';
+
+type PanelProps = {};
+
+const FunctionsSVG = () => (
```

Comment:
can we save the svg in the images/svg?

Reply:
oh lol, seems like I did add the image there but used a different one :/